source/_docs/guides/jira-integration.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Jira Integration
+description: How to connect Atlassian Jira issue tracking to a Drupal or WordPress site on Pantheon.
+tags: [siteintegrations]
+type: guide
+permalink: docs/guides/:basename/
+date: 5/4/2017
+contributors: scottmassey
+---
+
+Atlassian's Jira issue tracking is one of the most common applications used to manage projects for application development teams. It is part of a larger suite of tools which includes Bitbucket, a code repository, and Hipchat, a team messaging and collaboration application. Jira is extremely customizable, through manual configuration or the use of installable plugins. It allows for integration with tools in the Atlassian suite as well as other common development tools.
+
+In this guide, we will connect a Jira instance to a site on Pantheon. When changes are pushed to Pantheon that reference a jira issue ID, the commit message will appear in the Jira issue's activity log.
+
+
+## What You’ll Need
+
+- A Jira Instance.
```

Comment:
Maybe include a link to the recommended documentation on setting up a Jira instance.